Formula/qt5.rb

```diff
@@ -128,6 +130,15 @@ def install
     args << "-plugin-sql-mysql" if build.with? "mysql"
     args << "-plugin-sql-psql" if build.with? "postgresql"
 
+    # Both mysql and postgresql require openssl libs to link
+    if build.with? "openssl" or build.with? "mysql" or build.with? "postgresql"
+      openssl_opt = Formula["openssl"].opt_prefix
```

Comment:
You can make this
```
openssl = Formula["openssl"]
```
And then use `#{openssl.opt_lib}` and `#{openssl.opt_include}` below.